components/patient-access-rules/api/src/main/resources/ApplicationResources_es.properties

```diff
@@ -85,7 +85,7 @@ phenotips.homepage.sharedPatients.noData=No hay datos se ha compartido con usted
 
 phenotips.homepage.ownedFamilies.title=Mis familias
 phenotips.homepage.ownedFamilies.create=Registrar una nueva familia
-phenotips.homepage.ownedFamilies.noData=No eres el due\u00f1o de ninguno de los registros familiares.
+phenotips.homepage.ownedFamilies.noData=No eres el due\u00f1o de ninguno de los registros de familia.
```

Comment:
Out of habit I switched from the formal "es" version to the informal "eres". I'll make the fixes.